contacts.html

```diff
@@ -1 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
 
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./assets/css/contact.css">
+    <script src="./assets/js/firebase-connect.js"></script>
+    <script src="./assets/js/create-contact.js"></script>
+    <title>Contacts</title>
+</head>
+
+<body>
+
+    <main>
+        <div class="add-contact-wrapper">
+            <div class="add-contact-container">
+                <div class="left-side">
+                    <img src="./assets/img/logo-white.svg" alt="logo">
+                    <h2>Add contact</h2>
+                    <span>Task are better with a team!</span>
+                    <span class="border-line"></span>
+
+                </div>
+                <div class="right-side">
+                    <div class="person-img-container">
+                        <div class="person-img">
+                            <img src="./assets/img/person.svg" alt="person">
+                        </div>
+                    </div>
+                    <div class="form-elements">
+                        <div class="close-button-container">
+                            <img src="./assets/img/close.svg" alt="close">
+                        </div>
+
```

Comment:
sollten die Input-Felder vielleicht besser noch von einer <form> umschlossen werden? Oder ist das unnötig?

Reply:
Der focus/active-State der Inputs sollte noch eine blaue Umrandung haben, wie in Figma: 
![image](https://github.com/user-attachments/assets/4be779c3-d5de-4a9c-b358-ea001f9d30d5)

Reply:
ich habe gemacht